src/main/java/core/basesyntax/DepictionFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DepictionFigure {
```

Comment:
remove this unused interface

Reply:
I removed it in IDEA, how i must do it correctly?